src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -6531,4 +6531,30 @@ public override void Accept(OperationVisitor visitor)
             return visitor.VisitCaughtException(this, argument);
         }
     }
+
+    internal sealed class StaticLocalInitializationSemaphoreOperation : Operation, IStaticLocalInitializationSemaphoreOperation
+    {
+        public StaticLocalInitializationSemaphoreOperation(ILocalSymbol local, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue, bool isImplicit) :
+            base(OperationKind.StaticLocalInitializationSemaphore, semanticModel: null, syntax, type, constantValue, isImplicit)
```

Comment:
Are `constantValue` and `isImplicit` necessary, or are the values always `null` and `true`?

Reply:
They're always the same, I'll remove the parameters.

---
In reply to: [175159578](https://github.com/dotnet/roslyn/pull/25458#discussion_r175159578) [](ancestors = 175159578)

Reply:
Good catch, they should have been `default` and true. I always miss that `null` can be given to an `Optional` and not do what I would have expected it to.

---
In reply to: [175165195](https://github.com/dotnet/roslyn/pull/25458#discussion_r175165195) [](ancestors = 175165195,175159578)